modules/apps/portal-scheduler/portal-scheduler/src/main/java/com/liferay/portal/scheduler/internal/SchedulerEngineHelperImpl.java

```diff
@@ -547,13 +548,24 @@ public SchedulerJobConfiguration addingService(
 			Trigger trigger = null;
 
 			if (Validator.isNotNull(triggerConfiguration.getCronExpression())) {
+				if (StringUtil.equals(
+						triggerConfiguration.getCronExpression(),
+						StringPool.BLANK)) {
+
+					return null;
+				}
+
```

Comment:
@marianoalvarosaiz This part of code is dead code, you will never reach to this as `if (Validator.isNotNull(triggerConfiguration.getCronExpression()))` also checks empty string.

Please also take a look at your test why it did not catch this.

Thanks.
Tina.